core/rtw_recv.c

```diff
@@ -4193,8 +4202,17 @@ _func_exit_;
 }
 
 #ifdef CONFIG_NEW_SIGNAL_STAT_PROCESS
-void rtw_signal_stat_timer_hdl(RTW_TIMER_HDL_ARGS){
-	_adapter *adapter = (_adapter *)FunctionContext;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 15, 0)
```

Comment:
It's hard to read, maybe is better a single 
#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 15, 0)